pitch.md

```diff
@@ -1,60 +1,142 @@
+complicated semantics that doesn't require a lot of language design)?_  
+As my idea looks right now, the language itself is rather simple from a systems aspect
+because it's just rearranging the given lists, with some small optimizations and
+tasks. Because of this, I think most of the project would be refining the language
+design aspects of the projects so that it's very usable and intuitive. I think the
+main part of this project will be trying to mask that this is programming to the user
+and instead make it seem that they are just writing tasks.
 
 
 ## Scope
-_How big an idea is this? How ambitious is this project?_
+_How big an idea is this? How ambitious is this project?_  
+This started out as a very broad idea in scope, but I think that this iteration is
+much more refined and focused. Initially, I wanted to have this work for all to-do
+lists, regardless of the number of people or how people wanted to write their to-do
+lists. I think that the constraints will actually help people because each line of the
+program must take one of three forms: a list of times, a category name, or a task with
+a time for when it's due. Even if I worked to add more features (depending on how the
+first iteration of building this goes), I would still want it to be very simple because
+I think that will help users who do not have the background in computer science.
 
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
-off track, and how will you get the project back on track if needed?_
+off track, and how will you get the project back on track if needed?_  
+This will be a good projects because it will fit our week-by-week based development.
+This is a project that will be very easy to get done in chunks, starting with just
+the bare bones of the project and then building up the syntactic sugar that makes it
+easy to use and more features depending on the amount of time each part takes. I think
+some design issues will come up in trying to figure out if there is a universally simple
+way to give to-do lists as programs that everyone, especially non-programmers, will
+understand. There will probably also be some coding problems of figuring out how
+I want the IR to look (and how many levels of IR I want).  
+
+One way that I will know that this project has gotten out of hand is when the syntax
+has gotten out of hand. As I mentioned in the syntax discussion, right now I figure
+that the syntax will be rather simple. At first, it may be complicated because the
+sugar will not be my main prority. However, after adding sugar it will be a sign
+that I have added too many features when the syntax becomes too cluttered and hard to
+read. At that point, I will want to figure out what features to remove to make the syntax
+clean and easy to read and write.
```

Comment:
I'll put general comments  from talking with you and thinking about review questions here.
It seems like an awesome project idea since todo lists are so universally used.  Additionally it seems like a great take on how to make todo lists that resonates with the way I think. I think it is a little bit of a tough domain in the sense that there are lots of apps that do todo lists, but I think the category focus gives you a unique take on the task.  I think the biggest challenge will be making it feel easy to use.  Will users have to re run the program each time they add something to a todo list? This seems like it could be a little tedious if it's a todo list they update frequently.  On the other hand if it's a todo list that is something like class due dates it might be relatively static and this wouldn't be a problem.  I think the task of coming up with class due date todo lists is a great example of where your DSL is super useful, since it is easiest (at least for me) to think of due dates by class rather than by date but in the end I need to know them by date.  The project seems clearly defined and the scope sounds reasonable to me.